packages/service-clients/end-to-end-tests/azure-client/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @fluidframework/azure-end-to-end-tests
 
+## 2.62.0
+
+### Minor Changes
```

Comment:
Similar to Craig's suggestion in another comment, this it not a change in the API of this package, this one is just a consumer, so this note shouldn't be here and this can be a "Dependency updates only." note.

Reply:
@alexvy86 I have another PR out as @CraigMacomber suggested to do it another PR.
https://github.com/microsoft/FluidFramework/pull/25577